Source/WebCore/page/TextIndicator.h

```diff
@@ -167,6 +167,9 @@ class TextIndicator : public RefCounted<TextIndicator> {
     TextIndicatorPresentationTransition presentationTransition() const { return m_data.presentationTransition; }
     void setPresentationTransition(TextIndicatorPresentationTransition transition) { m_data.presentationTransition = transition; }
 
+    bool wantsBounce();
+    bool wantsManualAnimation();
```

Comment:
```suggestion
    bool wantsBounce() const;
    bool wantsManualAnimation() const;
```

Reply:
I think you need to `WEBCORE_EXPORT` `wantsManualAnimation` as well, since you call it from WK2 on iOS.